npm/webpack-dev-server/src/makeDefaultWebpackConfig.ts

```diff
@@ -37,9 +37,9 @@ export function makeDefaultWebpackConfig (
     },
     plugins: [
       new HtmlWebpackPlugin({
-        // Todo: Add indexHtmlFile when it gets added as a config property
         template: indexHtmlFile,
-      }) as any,
+        ...(config.devServerConfig.framework === 'angular' ? { scriptLoading: 'module' } : {}),
```

Comment:
Angular generates all of it's scripts with `<script type="module">`. Without this option, live-reloading breaks.